docs/Tutorial_Contrast_Profile2.ipynb

```diff
@@ -0,0 +1,485 @@
+{
```

Comment:
<p>I don't think it is correct to call the grey areas "anomalies". Are they ever referred to as anomalies in the paper?</p><p> </p><p>Edit: Below, you use the term "phenomena" and I think this is reasonable and suitable instead of "anomaly". </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/800/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>